pipeline-runner/R/gem2s-helpers.R

```diff
@@ -24,12 +24,12 @@ print_config <- function(step_number,step_name,input,pipeline_config,config){
   message(input)
   message("Pipeline config")
   message(pipeline_config)
-  print("Current working directory:")
-  print(getwd())
-  print("Experiment folder status:")
-  print(list.files("/input", all.files = TRUE, full.names = TRUE, recursive = TRUE))
-  print("Current config (meta.json):")
-  print(config)
+  message("Current working directory:")
+  message(getwd())
+  message("Experiment folder status:")
+  message(list.files("/input", all.files = TRUE, full.names = TRUE, recursive = TRUE))
+  message("Current config (meta.json):")
+  message(config)
```

Comment:
```suggestion
  message("Current working directory:")
  message(getwd())
  message("Experiment folder status:")
  print(list.files("/input", all.files = TRUE, full.names = TRUE, recursive = TRUE))
  message("Current config (meta.json):")
  print(config)
```
vectors and lists are one exception because it `paste0`'s them together:

![image](https://user-images.githubusercontent.com/15719520/123880580-d3640000-d8f7-11eb-873c-db1187d3d6fe.png)